### PR TITLE
Update kpod rm to use new container state

### PR DESCRIPTION
**Pull request by umohnani8**
kpod rm now uses the new container state and runtime

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @mheon PTAL

**Comment by rhatdan** [comment]
LGTM

**Comment by mheon** [comment]
Force is broken right now (we don't have the ability to stop containers yet, so we can't remove running containers). But otherwise this should work

File: cmd/kpod/rm.go [hunk 1]
```diff
@@ -30,40 +28,31 @@ var (
...
 
-	config, err := getConfig(c)
-	if err != nil {
-		return errors.Wrapf(err, "could not get config")
-	}
-	server, err := libkpod.New(config)
+	runtime, err := getRuntime(c)
 	if err != nil {
-		return errors.Wrapf(err, "could not get container server")
+		return errors.Wrapf(err, "Could not get runtime")
```
Comment (mheon) [inline cmd/kpod/rm.go]: Lowercase the C in could

**Comment by mheon** [comment]
LGTM, tests are passing

@rh-atomic-bot r+

**Pull request closed without merging**